<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DRuGLORD" failures="9" tests="16" name="com.cab.cucumber.runners.Runner" time="51.262" errors="0" timestamp="2025-07-28T13:57:50 IST" skipped="0">
  <testcase classname="com.cab.cucumber.runners.Runner" name="runScenario" time="3.099">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.cab.cucumber.definitions.Booking.verify_the_invalid_data(Booking.java:79)
at ✽.verify the invalid data(file:///C:/Users/hp/eclipse-workspace/AgileQADelivery-TTP1QESA2520-Group5/src/test/java/com/cab/cucumber/features/Booking.feature:19)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="com.cab.cucumber.runners.Runner" name="runScenario" time="2.612"/>
  <testcase classname="com.cab.cucumber.runners.Runner" name="runScenario" time="2.955">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.cab.cucumber.definitions.Booking.verify_the_invalid_data(Booking.java:79)
at ✽.verify the invalid data(file:///C:/Users/hp/eclipse-workspace/AgileQADelivery-TTP1QESA2520-Group5/src/test/java/com/cab/cucumber/features/Booking.feature:19)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="com.cab.cucumber.runners.Runner" name="runScenario" time="3.878"/>
  <testcase classname="com.cab.cucumber.runners.Runner" name="runScenario" time="2.224"/>
  <testcase classname="com.cab.cucumber.runners.Runner" name="runScenario" time="3.089"/>
  <testcase classname="com.cab.cucumber.runners.Runner" name="runScenario" time="2.925">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.cab.cucumber.definitions.Booking.verify_the_invalid_data(Booking.java:79)
at ✽.verify the invalid data(file:///C:/Users/hp/eclipse-workspace/AgileQADelivery-TTP1QESA2520-Group5/src/test/java/com/cab/cucumber/features/Booking.feature:19)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="com.cab.cucumber.runners.Runner" name="runScenario" time="3.138">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.cab.cucumber.definitions.Booking.verify_the_invalid_data(Booking.java:79)
at ✽.verify the invalid data(file:///C:/Users/hp/eclipse-workspace/AgileQADelivery-TTP1QESA2520-Group5/src/test/java/com/cab/cucumber/features/Booking.feature:19)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="com.cab.cucumber.runners.Runner" name="runScenario" time="2.306"/>
  <testcase classname="com.cab.cucumber.runners.Runner" name="runScenario" time="3.048">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.cab.cucumber.definitions.Booking.verify_the_invalid_data(Booking.java:79)
at ✽.verify the invalid data(file:///C:/Users/hp/eclipse-workspace/AgileQADelivery-TTP1QESA2520-Group5/src/test/java/com/cab/cucumber/features/Booking.feature:19)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="com.cab.cucumber.runners.Runner" name="runScenario" time="2.411"/>
  <testcase classname="com.cab.cucumber.runners.Runner" name="runScenario" time="3.114">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.cab.cucumber.definitions.Booking.verify_the_invalid_data(Booking.java:79)
at ✽.verify the invalid data(file:///C:/Users/hp/eclipse-workspace/AgileQADelivery-TTP1QESA2520-Group5/src/test/java/com/cab/cucumber/features/Booking.feature:19)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="com.cab.cucumber.runners.Runner" name="runScenario" time="3.062">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.cab.cucumber.definitions.Booking.verify_the_invalid_data(Booking.java:79)
at ✽.verify the invalid data(file:///C:/Users/hp/eclipse-workspace/AgileQADelivery-TTP1QESA2520-Group5/src/test/java/com/cab/cucumber/features/Booking.feature:19)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="com.cab.cucumber.runners.Runner" name="runScenario" time="7.026"/>
  <testcase classname="com.cab.cucumber.runners.Runner" name="runScenario" time="3.007">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.cab.cucumber.definitions.Booking.verify_the_invalid_data(Booking.java:79)
at ✽.verify the invalid data(file:///C:/Users/hp/eclipse-workspace/AgileQADelivery-TTP1QESA2520-Group5/src/test/java/com/cab/cucumber/features/Booking.feature:19)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="com.cab.cucumber.runners.Runner" name="runScenario" time="3.368">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.cab.cucumber.definitions.Booking.verify_the_invalid_data(Booking.java:79)
at ✽.verify the invalid data(file:///C:/Users/hp/eclipse-workspace/AgileQADelivery-TTP1QESA2520-Group5/src/test/java/com/cab/cucumber/features/Booking.feature:19)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- com.cab.cucumber.runners.Runner -->
